application/app/src/test/java/uk/nhs/digital/uec/api/integration/fuzzysearch/ElasticSearchTest.java

```diff
@@ -16,6 +17,7 @@
 import uk.nhs.digital.uec.api.util.PropertySourceResolver;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@Disabled
```

Comment:
were these tests stopped for development purposes or will we not want these tests any more ?

Reply:
These changes were already merged, this branch was rebased from master